db/migrate/20160830233558_migrate_ports_to_port_ranges_in_load_balancer_listener.rb

```diff
@@ -0,0 +1,26 @@
+class MigratePortsToPortRangesInLoadBalancerListener < ActiveRecord::Migration[5.0]
+  class LoadBalancerListener < ActiveRecord::Base
```

Comment:
@selmanj if there is column 'type'  in table, we also need `self.inheritance_column = :_type_disabled`
so:
```ruby
class LoadBalancerListener < ActiveRecord::Base
   self.inheritance_column = :_type_disabled
end
```